integrations/pom.xml

```diff
@@ -13,7 +13,8 @@
 
   <properties>
     <maven.deploy.skip>true</maven.deploy.skip> <!-- no need to be deployed during release, this is a test-only module -->
-    <jackson.version>2.12.4</jackson.version>
+    <jackson.version>2.13.1</jackson.version>
+    <enforcer.skip>true</enforcer.skip> <!-- I give up, too much effort to maintain enforcer here, should probably try clear out all enforcer only deps and see if it works -->
```

Comment:
😅 I had actually managed to get rid of it 😭
This is only a PITA so feel free to leave it.

Reply:
it just means dependabot PRs fail all the time and such a maintenance PITA